chemprop/v2/featurizers/molgraph/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .molgraph import MolGraph
 from .molecule import MoleculeMolGraphFeaturizerProto, MoleculeMolGraphFeaturizer
-from .reaction import RxnMolGraphFeaturizerProto, CondensedGraphOfReactionFeaturizer, RxnMode
+from .reaction import RxnMolGraphFeaturizerProto, CondensedGraphOfReactionFeaturizer, CGRFeaturizer, RxnMode
```

Comment:
Are `CondensedGraphOfReactionFeaturizer` and `CGRFeaturizer` not the same thing?

Reply:
They are aliases, so this just imports both of them to the top level of the subpacakage